Code.gs

```diff
@@ -151,16 +151,17 @@ function startSync(){
 
     targetCalendarName = calendar[0];
     var sourceCalendarURLs = calendar[1];
+    var colorId = calendar[2];
     var vevents;
 
     //------------------------ Fetch URL items ------------------------
     var responses = fetchSourceCalendars(sourceCalendarURLs);
     Logger.log("Syncing " + responses.length + " calendars to " + targetCalendarName);
     
     //------------------------ Get target calendar information------------------------
-    var targetCalendar = setupTargetCalendar(targetCalendarName);
+    var targetCalendar = setupTargetCalendar(targetCalendarName, colorId);
     targetCalendarId = targetCalendar.id;
-    Logger.log("Working on calendar: " + targetCalendarId);
+    Logger.log("Working on calendar: " + targetCalendarId + " with colorId: " + colorIds[colorId]);
```

Comment:
Similar to above comment, let's only log "with colorId..." if a color is specified

Reply:
Made it conditional now.